datalogistik/dataset.py

```diff
@@ -509,12 +515,24 @@ def convert(self, new_dataset):
         return new_dataset
 
     def output_result(self):
-        # TODO: we should also probably include dims here too
         output = {"name": self.name, "format": self.format}
 
         tables = {}
         for table in self.tables:
-            tables[table.table] = str(self.ensure_table_loc(table))
+            table_entry = {}
+            if table.dim:
+                table_entry["dim"] = table.dim
+            table_entry["path"] = str(self.ensure_table_loc(table))
+            tables[table.table] = table_entry
```

Comment:
This is slightly more simple (it also always has `dim` which IMO is good for an API like this since other things will parse this output and then we can always rely on it being there

```suggestion
            tables[table.table] = {
                "dim":  table.dim, # might need if table.dim after this to set to None
                "path": str(self.ensure_table_loc(table))
            }
```